app.css

```
@@ -0,0 +1,80 @@
+nav { font: 20px Copperplate; }
```

Try to write code more consistent: In this line, you write code in one line but most of your code wrote in another way
```suggestion
nav { 
    font: 20px Copperplate;
}
```